src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 #![crate_type = "lib"]
 #![cfg_attr(test, feature(test))]
-#![recursion_limit = "200"]
+#![recursion_limit = "400"]
```

Comment:
Why change to 400?

Reply:
More content in `try_stream!` macro requires a larger limit.